client/src/test/helper.ts

```diff
@@ -11,14 +10,29 @@ export async function sleep(ms: number): Promise<NodeJS.Timeout> {
   return new Promise((resolve) => setTimeout(resolve, ms))
 }
 
+/**
+ * Try requiring
+ */
+function tryRequire(path: string): any {
+  try {
+    return require(path)
+  } catch (err) {
+    return {}
+  }
+}
+
 /**
  * Activates the vscode.prisma extension
  * @todo check readiness of the server instead of timeout
  */
 export async function activate(docUri: vscode.Uri): Promise<void> {
   // The extensionId is `publisher.name` from package.json
-  const pj = require('../../../package.json')
-  const ext = vscode.extensions.getExtension(pj.publisher + '.' + pj.name)!
+  const pj = tryRequire('../../../package.json')
+  const ext = vscode.extensions.getExtension(pj.publisher + '.' + pj.name)
```

Comment:
`tryRequire` can return an empty object. In that case, is this function call handled properly?

`vscode.extensions.getExtension(undefined + '.' + undefined)`

Reply:
good catch, I changed that